packages/icons/src/metadata.ts

```diff
@@ -0,0 +1,1816 @@
+/**
+ * Metadata for all available onyx icons.
+ */
+export const ICON_METADATA = {
```

Comment:
question: how is this list going to be maintained?

Reply:
If UX adds a new icon, they should also add a metadata entry here.
I documented the contribution flow in the README :)